rumqttc/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `MqttOptions::set_request_modifier` for setting a handler for modifying a websocket request before sending it.
 
 ### Changed
+- Removed dependency vulnerability, see [rustsec](https://rustsec.org/advisories/RUSTSEC-2023-0065). Update of `tungstenite` dependency.
```

Comment:
Can we move this entry under `### Security` section please, so it would follow previous changes regarding webpki?